firebase_admin/ml.py

```diff
@@ -592,6 +601,36 @@ def as_dict(self, for_upload=False):
         return {'gcsTfliteUri': self._gcs_tflite_uri}
 
 
+class TFLiteAutoMlSource(TFLiteModelSource):
+    """TFLite model source representing a tflite model created via AutoML."""
+
+    def __init__(self, auto_ml_model, app=None):
+        self._app = app
+        self._auto_ml_model = _validate_auto_ml_model(auto_ml_model)
+
+    def __eq__(self, other):
+        if isinstance(other, self.__class__):
+            return self._auto_ml_model == other._auto_ml_model  # pylint: disable=protected-access
```

Comment:
Just access the public property and get rid of the linter directive.

Reply:
Done